go.mod

```diff
@@ -4,8 +4,8 @@ go 1.19
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.10.0
-	github.com/ava-labs/avalanche-network-runner v1.7.1
-	github.com/ava-labs/avalanchego v1.10.8
+	github.com/ava-labs/avalanche-network-runner v1.7.2-0.20230821200536-177d9acbae64
+	github.com/ava-labs/avalanchego v1.10.9-rc.6
```

Comment:
why we used rc?

Reply:
I created this before the release was out. I'll bump this to v1.10.9.

Reply:
Once there is an ANR release compatible with v1.10.9, I'll update the dependency.